builder/azure/dtl/config.go

```diff
@@ -278,6 +279,9 @@ type Config struct {
 	// DisallowPublicIPAddress - Indicates whether the virtual machine is to be created without a public IP address.
 	DisallowPublicIP bool `mapstructure:"disallow_public_ip" required:"false"`
 
+	// SysPrepDone - Indicates whether SysPrep is to be requested to the DTL or has been already applied.
+	SysPrepDone bool `mapstructure:"sysprep_done" required:"false"`
```

Comment:
I found the name for this configuration to be confusing. From the description I take it to mean that if set to true the source image already had sysprep applied to it so there is no need to rerun again. Is that correct? 
```go
	// SkipSysprep - Indicates whether SysPrep is to be requested to the DTL or if it should be skipped because it has already been applied. Defaults to false.
	SkipSysprep bool `mapstructure:"skip_sysprep" required:"false"`
```

If my understanding is correct, I would suggest calling this argument "skip_sysprep"

Reply:
good point! done!